catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -361,7 +361,7 @@ public void setSceneToPlay(Scene scene) {
 	}
 
 	public Scene getCurrentScene() {
-		if (currentScene == null) {
+		if (currentScene == null && project != null) {
 			currentScene = project.getDefaultScene();
 		}
 		return currentScene;
```

Comment:
Well, I've experienced in general some crashes concerning scenes - not only in this branch. This was one of the crashes. But it has nothing to do with userbricks, so I've reverted the change.